consensus/istanbul/backend/engine.go

```diff
@@ -289,8 +289,8 @@ func (sb *backend) verifyCommittedSeals(chain consensus.ChainReader, header *typ
 		}
 	}
 
-	// The length of validSeal should be larger than number of faulty node + 1
-	if validSeal < snap.ValSet.QuorumSize(validator.IBFT_FORMULA_CEIL_2N_3) {
+	// The validSeal number should be greater than equal to ceil(2N/3) i.e., quorum size   
+	if validSeal < snap.ValSet.QuorumSize() {
```

Comment:
I'm sorry I missed this before. For sealing, we still only need f+1. Can you revert these back?